src/main/java/com/google/devtools/build/lib/bazel/rules/android/android_ndk_build_file_template.txt

```diff
@@ -34,6 +36,46 @@ alias(
     actual = "%defaultCrosstool%",
 )
 
+# Platform definitions for Android.
+# The constraints are on OS and Architecture. Currently, there are
+# no constraints on API levels. There is also no constraint on the STL
+# as Android NDK ships with libcpp only.
+#
+# Keep consistent with Android NDK supported architectures.
+# See https://docs.bazel.build/versions/master/android-ndk.html#integration-with-platforms-and-toolchains
+# for more information.
+platform(
```

Comment:
It would be nice to have these in @bazel_platforms, but that would require an extra repository dependency. I don't know how well that works for Android rule users.

Reply:
Is the plan for `@bazel_platforms` to be added to every user's WORKSPACE file when the world moves to platforms?

Reply:
Bazel bundles @platforms (https://source.bazel.build/bazel/+/4ba404f7ed0473df3f0effa016c107ef677464f6:src/main/java/com/google/devtools/build/lib/bazel/repository/local_config_platform.WORKSPACE) but it's bundled in a way that allows users to override it with their own version. Because of the motivation to have flag-less builds and host platform autoconfiguration we cannot unbundle platforms from Bazel. So if you want to put these platforms into @platforms, you technically can without extra dependency.

But I'm not sure if we should? I didn't expect actual 'platform' targets to be added to @platforms. The best place for android specific platforms sounds like `rules_android`. John can you elaborate on this a little?